### PR TITLE
WINTER IN THE SUMMER FEATURE FREEZE 2016 (part 2)

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Until June 15th. 

No balance memes or new features during the freeze. Redoing sprites, refactoring code, or updating to tgui is fine. Most importantly though go get the issue count lowered. If we hit 650 issues the freeze ends early and we can get back to ruining the game.

SUBSYSTEM/MC code is also frozen as per MSO's request - <B>Remie</B>

### CODE REVIEW

**Comment by PKPenguin321** [comment]
Reeeeeeeeee

File: code/__DEFINES/math.dm [hunk 1]
```diff
@@ -8,4 +8,4 @@
 #define ONE_ATMOSPHERE			101.325	//kPa
 #define T0C						273.15	// 0degC
 #define T20C					293.15	// 20degC
-#define TCMB					2.7		// -270.3degC
+#define TCMB					really cold		// -270.3degC
```
Comment (PKPenguin321) [inline code/__DEFINES/math.dm]: ❄⛄

**Comment by lzimann** [comment]
not enough tags

**Comment by kevinz000** [comment]
You're an issue :^]

File: code/__DEFINES/math.dm (hunk 1 above)
Comment (Bawhoppen) [inline code/__DEFINES/math.dm]: :shaved_ice:

**Comment by Iamgoofball** [comment]
@KorPhaeron we only have 672 actual bugs

https://github.com/tgstation/-tg-station/issues?q=is%3Aissue+is%3Aopen+label%3ABug

**Comment by Jalleo** [comment]
@Iamgoofball there are a lot of things that need fixing that arebt tagged as bugs.

**Comment by Bawhoppen** [comment]
we can easily compare the current times of tg to that of the roman republic, kor is julius caesar and you can see the decay of the republican ideals, kor adds his own rainbow meme tags as opposed to the tradition of another maintainer which is similar to the classic traditions of the roman senate's being stripped away and the decline of the republic, soon we will have to stab kor 23 times

**Comment by Bawhoppen** [comment]
oh hey he got the bot working after all

**Comment by bgobandit** [comment]
no ayy lmao tag

**Comment by KorPhaeron** [comment]
Someone deleted that tag at some point, I put every tag we had

**Comment by KorPhaeron** [comment]
okay this is lame so new plan

**Comment by Bawhoppen** [comment]
:tada: :tada: :tada: :tada: :tada: :tada: :tada: :tada: :tada: :tada: :tada: :tada: :tada:

**Pull request closed without merging**